x-pack/plugins/infra/public/utils/logs_overview_fetchers.ts

```diff
@@ -89,9 +89,8 @@ async function fetchLogsOverview(
   params: FetchDataParams,
   dataPlugin: InfraClientStartDeps['data']
 ): Promise<StatsAndSeries> {
-  const esSearcher = dataPlugin.search.getSearchStrategy('es');
   return new Promise((resolve, reject) => {
-    esSearcher
+    dataPlugin.search
```

Comment:
Is there some documentation for how to interact with this API? How can I confirm that `dataPlugin.search.search` is the correct API function etc?

Reply:
The docs are in the making. 
Synced with @weltenwort to confirm the changes suit your needs.

Reply:
OK, in the meantime without docs, can you confirm that the API is definitely `dataPlugin.search.search` vs `dataPlugin.search`? And what are the list of available keys and methods on `dataPlugin`?

Reply:
Data plugin contains several services 
 - query (everything that has to do with the search bar - time range, query string, filters)
 - search (which contains the `search` function and the `aggs` helpers
 - index patterns
 - field formatting
 - autocompletion

The method relevant to you is indeed `dataStart.search.search()`